graph/src/main.py

```diff
@@ -15,22 +15,25 @@
 @app.route('/data')
 @cross_origin()
 def data_index():
-    my_output = json.loads(input_data)
+    content = get_json_content(os.path.join(ui_static_content, "fake_graph.json"))
```

Comment:
how about keeping the original mechanism and piping the data via additional Make target? Or using default data when no data is provided?